src/app/landing/PartnerPicture.jsx

```diff
@@ -1,12 +1,38 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import './PartnerPicture.css';
 
-const PartnerPicture = ({ src, alt, link }) => (
+const PartnerPicture = ({ imagePartialPath, alt = '', link }) => (
   <div className="PartnerPicture">
     <a href={link}>
-      <img src={src} alt={alt} />
+      <picture>
+        <source
+          className="OnboardingImage"
```

Comment:
I thought className only needed to be set on the `img` element, no?
And if not, don't Testimonial.jsx and all the pictures in LandingPage.jsx also need the className on the `source`s as well?

Reply:
Yeah, they’re only needed for img elements, removed